test/infrastructure/docker/docker/machine.go

```diff
@@ -325,18 +330,16 @@ func (m *Machine) ExecBootstrap(ctx context.Context, data string) error {
 		return errors.Wrap(err, "failed to join a control plane node with kubeadm")
 	}
 
-	var outErr bytes.Buffer
 	var outStd bytes.Buffer
 	for _, command := range commands {
 		cmd := m.container.Commander.Command(command.Cmd, command.Args...)
```

Comment:
Just out of curiosity: why don't we set stderr anymore?

Reply:
When calling the CLI, the stderr output was usually the errors from calling the command. Now we get that type of error output from the actual `error` returned from the API calls. So it did not seem necessary to separate those out anymore.

Looking at this again, there may be stderr that could possibly be useful in the logging call below. I'll add that back in, and we can decide at a later point if it's not needed or not.